PREINSTALL.md

```diff
@@ -17,16 +17,19 @@ extension. You can do so on the [Algolia](https://www.algolia.com/) site.
 
 #### Billing
 
-To install an extension, your project must be on the
-[Blaze (pay as you go) plan][blaze-pricing].
+This extension uses the following Firebase services which may have associated charges:
 
-- You will be charged [around $0.01 per month][pricing-examples] for each
-  instance of this extension you install.
-- This extension uses other Firebase and Google Cloud Platform services,
-  which have associated charges if you exceed the service's free tier:
-    - Cloud Functions (Node.js 10+ runtime. [See FAQs][faq].)
-    - Cloud Firestore
+- Cloud Firestore
+- Cloud Functions
 
-[blaze-pricing]: https://firebase.google.com/pricing
-[pricing-examples]: https://cloud.google.com/functions/pricing#pricing_examples
-[faq]: https://firebase.google.com/support/faq#expandable-24
+This extension also uses the following third-party services:
+
+- Algolia Billing ([pricing information](https://www.algolia.com/pricing))
```

Comment:
Just "Algolia", not "Algolia Billing", right?

Reply:
right.  i will correct it.